k8s/upgrade/src/plugin/upgrade.rs

```diff
@@ -103,25 +103,66 @@ pub struct UpgradeArgs {
     /// Upgrade to an unsupported version.
     #[clap(global = true, hide = true, long, default_value_t = false)]
     pub skip_upgrade_path_validation_for_unsupported_version: bool,
+
+    /// Upgrade to an unsupported version.
```

Comment:
```suggestion
    /// The set values specified by the user for upgrade.
    /// (can specify multiple or separate values with commas: key1=val1,key2=val2).
```

Reply:
Helm seems to have this:
```
      --set stringArray                            set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)
      --set-file stringArray                       set values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2)
      --set-json stringArray                       set JSON values on the command line (can specify multiple or separate values with commas: key1=jsonval1,key2=jsonval2)
      --set-string stringArray                     set STRING values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)
```
Should we support more of this? IMHO I think we should at least support set-file.
CC @niladrih

Reply:
`set-file` can be taken as separate PR